plugin/FormData.php

```diff
@@ -162,7 +163,7 @@ public function getSearchcriteriaData(): array
 			$inputConfigName = $pSearchcriteriaFields->getFieldNameOfInput($input);
 			$inputModule = $inputs[$inputConfigName] ?? null;
 
-			if (onOfficeSDK::MODULE_SEARCHCRITERIA === $inputModule) {
+			if (onOfficeSDK::MODULE_SEARCHCRITERIA === $inputModule || ($isGetMessageField && in_array($input, ['message']))) {
```

Comment:
I would say the `message` field is neither a search criteria field, nor address or estate. It is something special that is only supposed to be used in forms.